public/javascripts/SVLabel/src/SVLabel/canvas/ContextMenu.js

```diff
@@ -472,15 +472,38 @@ function ContextMenu (uiContextMenu) {
             _setTags(targetLabel);
             _setTagColor(targetLabel);
             if (getStatus('disableTagging')) { disableTagging(); }
+
+            // Hide the severity menu for the Pedestrian Signal label type.
+            if (labelType === 'Signal') {
+                $severityMenu.css({visibility: 'hidden', height: '0px'});
+            } else {
+                $severityMenu.css({visibility: 'inherit', height: '50px'});
+            }
             var windowHeight = $menuWindow.outerHeight();
+            var connectorBuffer = 3; // for connector to overlap border of label icon
+
+            var connectorHeight = parseInt(window.getComputedStyle($connector[0]).getPropertyValue("height"));
```

Comment:
You should keep the `parseInt` here and remove from all the places further down where you call it again on the already parsed variables! Might be enough to be able to fit some of those calculations that are split across multiple lines onto one line :grin: